### DISCUSSION TITLE
is the name change of revanced-manager on purpos

### DISCUSSION THREAD

**Discussion by yonggamer** [ReVanced/revanced-discussions#1087] [Question]
it went from
![png](https://github.com/user-attachments/assets/238b6b56-2da4-4d53-a39a-a293bb6f5c1e)
to
![Image](https://github.com/user-attachments/assets/9fff4d21-e944-45a2-a0e2-4594f39492b2)

**Comment by oSumAtrIX** [comment]
You are using an old release. Update.
